src/expectedMismatches.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Would be cleaner to add this to the token data files instead of storing this somewhere else.    The advantage of that over this method is this is one more thing that needs to be kept in sync if say the folder name changed

Reply:
nice, good suggestion